### PR TITLE
Cleanup bloat

### PR DESCRIPTION
**Pull request by olivernosto**
## Description
Remove styles from JS code, improve min height rendering on expanded tile, fix default settings

## Checklist

- [ ] Pull Request is properly described.
- [ ] The corresponding Jira ticket is updated.
- [ ] I have requested a review from at least one reviewer.
- [ ] The changes are tested
- [ ] I have verified my UX changes with a designer
- [ ] I have checked my code for any possible security vulnerabilities

### CODE REVIEW

File: widgets/styles/partials/_swiper-expanded.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
+$nosto-container-margin: 20px;
```
Comment (manikandan-ravikumar) [inline widgets/styles/partials/_swiper-expanded.scss:1]: Rename this to match the component name (_expanded-swiper.scss)

File: widgets/styles/partials/_swiper.scss [hunk 2]
```diff
@@ -0,0 +1,13 @@
+@forward "pkg:swiper/swiper-bundle.css";
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap');
+
+:host {
+    font-family: Inter, sans-serif;
+}
+
+@font-face {
```
Comment (manikandan-ravikumar) [inline widgets/styles/partials/_swiper.scss:8]: We do have font.scss partial already. Let's not spread fonts everywhere.

File: widgets/styles/templates/expanded-tiles/_index.scss [hunk 3]
```diff
@@ -109,6 +109,7 @@ expanded-tiles:not(:empty) {
       border-radius: var(--expanded-tile-border-radius);
       grid-template-columns: minmax(0, dimens.$panel-left-width-lg) minmax(0, dimens.$panel-right-width-lg);
       grid-template-rows: 1fr;
+      min-height: 80vh;
```
Comment (manikandan-ravikumar) [inline widgets/styles/templates/expanded-tiles/_index.scss:112]: Shouldn't we follow the height specified in the Figma design?

Comment (manikandan-ravikumar) [reply widgets/styles/templates/expanded-tiles/_index.scss:112]: Also, now the swiper adapts to content height automatically

File: widgets/styles/partials/_swiper-expanded.scss (hunk 1 above)
Comment (olivernosto) [reply widgets/styles/partials/_swiper-expanded.scss:1]: Resolved. Thanks!

File: widgets/styles/partials/_swiper.scss (hunk 2 above)
Comment (olivernosto) [reply widgets/styles/partials/_swiper.scss]: Fixed.

File: widgets/styles/templates/expanded-tiles/_index.scss (hunk 3 above)
Comment (olivernosto) [reply widgets/styles/templates/expanded-tiles/_index.scss]: Reverted. Thanks!

**Pull request merged**